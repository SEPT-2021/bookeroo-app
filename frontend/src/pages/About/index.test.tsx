import React from "react";
import ReactDOM from "react-dom";
import { render, cleanup } from "@testing-library/react";
import { QueryClient, QueryClientProvider } from "react-query";
import AboutUs from "./index";

afterEach(cleanup);
describe("Rendering page About page", () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(
      <QueryClientProvider client={new QueryClient()}>
        <AboutUs />
      </QueryClientProvider>,
      div
    );
    expect(div).toMatchSnapshot();
  });
});

describe("AboutUs matches snapshot", () => {
  test("render AboutUs", () => {
    const { container } = render(
      <QueryClientProvider client={new QueryClient()}>
        <AboutUs />
      </QueryClientProvider>
    );
    expect(container).toMatchSnapshot();
  });
});
