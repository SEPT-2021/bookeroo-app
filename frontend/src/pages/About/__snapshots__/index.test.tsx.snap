// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs matches snapshot render AboutUs 1`] = `
<div>
  <main
    class="MuiGrid-root AboutUs-root-1 MuiGrid-container"
  >
    <div
      class="row align-items-center my-5"
    >
      <div
        class="col-lg-7"
      >
        <img
          alt=""
          class="img-fluid rounded mb-4 mb-lg-0"
          src="logo.svg"
        />
      </div>
      <div
        class="col-lg-5"
      >
        <h1
          class="font-weight-light"
        >
          About Us
        </h1>
        <p>
          To be populated......
        </p>
      </div>
    </div>
  </main>
</div>
`;

exports[`Rendering page About page renders without crashing 1`] = `
<div>
  <main
    class="MuiGrid-root AboutUs-root-1 MuiGrid-container"
  >
    <div
      class="row align-items-center my-5"
    >
      <div
        class="col-lg-7"
      >
        <img
          alt=""
          class="img-fluid rounded mb-4 mb-lg-0"
          src="logo.svg"
        />
      </div>
      <div
        class="col-lg-5"
      >
        <h1
          class="font-weight-light"
        >
          About Us
        </h1>
        <p>
          To be populated......
        </p>
      </div>
    </div>
  </main>
</div>
`;
